vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt

```diff
@@ -68,16 +67,13 @@ import javax.inject.Inject
 import javax.inject.Singleton
 import kotlin.random.Random
 
-/**
- * Util class for creating notifications.
- * Note: Cannot inject ColorProvider in the constructor, because it requires an Activity
- */
```

Comment:
Always a bit sad to see comment blocks vanish.

Reply:
OK, this is old comment I think, you are right. Also I do not understand why this class has `@Singleton`. And yes this would definitely needs a renaming. This class is maybe 6 or 7 years old :).